src/overlays/actors/ovl_Demo_Geff/z_demo_geff.h

```diff
@@ -4,10 +4,18 @@
 #include <ultra64.h>
 #include <global.h>
 
+#include "../ovl_Demo_Gt/z_demo_gt.h"
```

Comment:
Include this in the C file instead, so whatever includes `z_demo_geff.h` doesn't also include `z_demo_gt.h` for no reason.

Reply:
We need the include here if we want 0x158 to be a DemoGt pointer. I think it's better than using Actor since we know it's DemoGt.

Reply:
Ah I missed that part, my bad.